platform-tests/src/test/resources/console/details/report/Report-reportMultiEntriesWithMultiMappings-tree-unicode.out.txt

```diff
@@ -0,0 +1,26 @@
+╷
+└─ JUnit Jupiter ✔
+   └─ Report ✔
+      └─ reportMultiEntriesWithMultiMappings(TestReporter) ✔
+            ....-..-..T..:...*
+               user name = `dk38`
+               award year = `1974`
+            ....-..-..T..:...* single = `mapping`
+            ....-..-..T..:...*
```

Comment:
How about an optional caption an user can provide to describe an entry set? In pseudo-code:

`TestReporter.publishEntry("user table", Map.of(...));`

which yields:

```
            user table [....-..-..T..:...*]
               user name = dk38
               award year = 1974
```